frontend/src/js/components/tenants/TenantPage.tsx

```diff
@@ -50,7 +50,7 @@ export const TenantPage = () => {
       <h2>Tenants</h2>
       {tenants.length ? <TenantList /> : <TenantsEmptyState openModal={() => setShowCreate(true)} />}
       <Chip color="primary" icon={<AddIcon />} label="Add tenant" onClick={() => setShowCreate(true)} />
-      <TenantCreateForm open={showCreate} onCloseClick={() => setShowCreate(false)} />
+      {showCreate && <TenantCreateForm open={showCreate} onCloseClick={() => setShowCreate(false)} />}
```

Comment:
doesn't the `open` prop handle the visibility of the form already?

Reply:
Yes, but it's an easy way to reset form state :sweat_smile: